contributors/devel/sig-architecture/api_changes.md

```diff
@@ -1016,6 +1026,155 @@ In future Kubernetes versions:
     }
     ```
 
+#### New enum value in existing field
+
+A developer is considering adding a new allowed enum value of `"OnlyOnTuesday"`
+to the following existing enum field:
+
+```go
+type Frobber struct {
+  // restartPolicy may be set to "Always" or "Never".
+  // Additional policies may be defined in the future.
+  // Unrecognized policies should be treated as "Never".
```

Comment:
(I might be missing something here) Does this line mean that when an unrecognized policy (`RestartPolicy: OnlyOnTuesday`) is specified in Release 1 with the feature gate disabled, the value of `RestartPolicy` will be set to `Never`?

If yes, should we avoid failing on validation when `RestartPolicy: OnlyOnTuesday` is specified to continue behaving in a situation where the new enum value does not exist?

Reply:
> Does this line mean that when an unrecognized policy (`RestartPolicy: OnlyOnTuesday`) is specified today (release prior to `Release 1`), validation will not fail and the value of `RestartPolicy` will be set as `Never`?

no, this is guidance for API clients encountering enum values they don't recognize when reading from the API. essentially, what should go in the `default` case of a switch statement:

```
switch obj.RestartPolicy {
case api.Always:
  ...
case api.Never:
  ...
default:
  ... ?
}
```

I'll try to clarify that.